stunnel/init.sls

```diff
@@ -27,6 +27,9 @@ stunnel_package:
     - user: {{ stunnel.root_user }}
     - group: {{ stunnel.group }}
     - mode: 750
+{%- if stunnel.clean_conf_dir %}
+    - clean: True
```

Comment:
this is setting the option for the `services.d` directory not the `conf.d` directory.

Reply:
Do you suggest to change the var name to clean_service_dir?

Reply:
That would make more sense to me.

Reply:
I agree, since the scope changed from your first version, `clean_services_dir` would be a better name for this setting now. Also, the explanation in `pillar.example` should be updated to say it's only going to clean the `services.d` folder.

Reply:
It has been updated